src/main/java/seedu/weme/logic/commands/exportcommand/ExportCommand.java

```diff
@@ -54,6 +65,17 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(MESSAGE_SUCCESS);
     }
 
+    public DirectoryPath getExportPath(Model model) throws CommandException {
```

Comment:
Might be more intuitive to just return a Path object here instead of a DirectoryPath object

Reply:
True but DirectoryPath does validation of the directory path so it is a security measure.

Reply:
if so then wouldnt it be better to just call the DirectoryPath's isValid method rather than converting the whole thing into a DirectoryPath object?

Reply:
It is not a utility class so we should minimize coupling. I will remove the staticness of the isValid method.

Reply:
All our data classes like Description have a static isValid method though, changing this is actually deviating from the original design. In this case since you are creating a DirectoryPath object, the coupling is the same